core-redirects-studio-lib/src/main/java/com/tallence/core/redirects/studio/repository/RedirectRepositoryImpl.java

```diff
@@ -151,7 +151,10 @@ public Pageable getRedirects(String siteId, String search, String sorter, String
     List<Redirect> redirects = new ArrayList<>();
     for (int i = (page - 1) * pageSize; i < page * pageSize; i++) {
       if (i < hits.size()) {
-        redirects.add(convertToRedirect(hits.get(i)));
+        Content content = hits.get(i);
+        if (!content.isDeleted()) {
```

Comment:
Das sollte vermutlich besser `isInProduction` testen:
```suggestion
        if (content.isInProduction()) {
```

Reply:
Ist umgesetzt.